TaskMonitorPlus/__init__.py

```diff
@@ -99,6 +99,21 @@ def __stop__(self):
         eg.messageReceiver.RemoveHandler(WM_APP + 2, self.WindowCreatedProc)
         eg.messageReceiver.RemoveHandler(WM_APP + 3, self.WindowDestroyedProc)
 
+    def CheckPlugins(self):
+        for plugin in eg.pluginList:
+            if (
+                plugin.info.guid == "{D1748551-C605-4423-B392-FB77E6842437}"
```

Comment:
look for the guid for the plugin which is contained in the plugininstanceinfo instance for the plugin (info)